gremlin-go/driver/resultSet.go

```diff
@@ -149,11 +149,15 @@ func (channelResultSet *channelResultSet) Channel() chan *Result {
 }
 
 // One returns the next Result from the channelResultSet, blocking until one is available.
```

Comment:
Update comment?

Reply:
added in https://github.com/apache/tinkerpop/pull/1653/commits/dcb281409d65d115fb960cc534954e84f7b5415d